src/storage/secondary/block/rle_block_builder.rs

```diff
@@ -166,7 +246,7 @@ mod tests {
             .iter()
             .cycle()
             .cloned()
-            .take(u16::MAX as usize * 2)
+            .take(u32::MAX as usize + 1)
```

Comment:
Will this cause test to be running very slow? If so, we can avoid testing it.

Reply:
I think it does cause test to be running very slow. But I don't know how to change the test myself.